frontend/src/components/CampaignFormBO.jsx

```diff
@@ -0,0 +1,237 @@
+            value={newCampaign.time_start}
+            onChange={event => {
+              setNewCampaign({
+                ...newCampaign,
+                time_start: event.target.value
+              });
+            }}
+            required
+          />
+        </div>
+
+        <div className="container">
+          <label>Campaign end date :</label>
+          <p>Please enter strict syntax</p>
+          <span>ex: "2020-03-29 00:00:00"</span>
+          <input
+            type="datetime"
+            placeholder="YYYY-MM-DD 00:00:00"
+            value={newCampaign.time_end}
+            onChange={event => {
+              setNewCampaign({
+                ...newCampaign,
+                time_end: event.target.value
+              });
+            }}
+            required
+          />
+        </div>
+
+        <div className="container">
+          <label>Campaign event date :</label>
+          <input
+            type="date"
+            value={newCampaign.date_event}
+            onChange={event => {
+              setNewCampaign({
+                ...newCampaign,
+                date_event: event.target.value
+              });
+            }}
+            required
+          />
+        </div>
+
+        <div className="container">
+          <label>First donation value :</label>
+          <input
+            type="number"
+            value={newCampaign.value1}
+            onChange={event => {
+              setNewCampaign({
+                ...newCampaign,
+                value1: parseInt(event.target.value)
+              });
+            }}
+            required
+          />
+        </div>
+
+        <div className="container">
+          <label>Second donation value :</label>
+          <input
+            type="number"
+            value={newCampaign.value2}
+            onChange={event => {
+              setNewCampaign({
+                ...newCampaign,
+                value2: parseInt(event.target.value)
+              });
+            }}
+            required
+          />
+        </div>
+
+        <div className="container">
+          <label>Third donation value :</label>
+          <input
+            type="number"
+            value={newCampaign.value3}
+            onChange={event => {
+              setNewCampaign({
+                ...newCampaign,
+                value3: parseInt(event.target.value)
+              });
+            }}
+            required
+          />
+        </div>
+
+        <div className="container">
+          <label>Id user :</label>
+          <input
+            type="number"
+            value={newCampaign.id_user}
+            maxLength="2"
+            onChange={event => {
+              setNewCampaign({
+                ...newCampaign,
+                id_user: parseInt(event.target.value)
+              });
+            }}
+          />
+        </div>
+
+        <div className="container">
+          <label>Id ambassador :</label>
```

Comment:
A `<select>` (with names) would be better

Reply:
made this one the next step, now would have to assemble it first.